libs/src/api/account.js

```diff
@@ -252,6 +253,15 @@ class Account extends Base {
     return this.identityService.checkIfEmailRegistered(email)
   }
 
+  /**
+   * Get the current user's email address
+   * @returns {{email: string | undefined | null}}
+   */
+  async getUserEmail () {
+    this.REQUIRES(Services.IDENTITY_SERVICE)
+    return this.identityService.getUserEmail()
```

Comment:
so only the typescript files get the special eslint config that says await promises, so not caught here, but maybe we should do it for consistency?

Reply:
actually this entire file follows this pattern so ill just leave it the way it is